### ISSUE TITLE
Not able to replicate results 

### ISSUE THREAD

**Issue by voidcf** [city-super/Octree-GS#35]
Hi, thanks for the great work!
Lately, I've been trying to replicate the results reported by Octree-GS in their paper for the Block_small scene, but the maximum PSNR I've gotten so far was **~18**. 
- I used the **block_small** scene by taking the files only from the aerial folder and the transforms.json inside it as mentioned [here ](https://github.com/city-super/Octree-GS/issues/34#issuecomment-2191391325) and [here](https://github.com/city-super/Octree-GS/issues/34#issuecomment-2191199417)
- I also made changes to the cuda rasterizer as suggested [here](https://github.com/city-super/Octree-GS/issues/16#issuecomment-2082231754) and recompiled it 
- The GPU that I am using is a Quadro RTX 6000 (24G)

Am I missing something here, please do let me know... I've attached the [log](https://github.com/user-attachments/files/16024618/outputs.log) file for your reference as well.

**Comment by jianglh-WHU** [comment]
can you show the poses of training images and the init point cloud?

**Comment by voidcf** [comment]
I've used the same point cloud that you mentioned [here](https://github.com/city-super/Octree-GS/issues/21#issuecomment-2100939298) and poses are from the transforms.json file in MatrixCity/aerial_street_fusion/aerial/transforms.json

**Comment by jianglh-WHU** [comment]
can you visualize it?

**Comment by voidcf** [comment]
#16, my point cloud is looking similar to this.

**Comment by jianglh-WHU** [comment]
In MatrixCity, we conducted an experiment that involved fusing data from both aerial and street views. To represent this data, we provided aerial_street_fusion, which is discussed in more detail in issue [#30](https://github.com/city-super/MatrixCity/issues/30). However, for the Octree-GS experiment, we utilized aerial images from the block_small region.
For your convenience, we have made available the [point cloud and pose](https://drive.google.com/drive/folders/1qhpyrzKp3H0rOxq7tZpCn0wVyt8FL0bJ?usp=sharing) for your use.
